code-challenges.js

```diff
@@ -1,62 +1,126 @@
-
-const currentCohort = "Foxtrot 2023"
+// name of function: competitions
+// input: 2 arrays of numbers
+// output: the amount of variables within both arrays combined
+// process: combine values within arrays and return the amount of characters of the combined values
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// const competitions = padres1984WorldSeriesRuns.concat.(padres1998WorldSeriesRuns){
+// }
+// console.log(competitions.length)
+// competitions.length
+var padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+var padres1998WorldSeriesRuns = [6, 3, 5, 3]
+console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+    let array = [array.length]
+//NOTES: I keep getting an error message, I think I'm on the right track but there's probably a lot wrong with my syntax?
+// // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+// const currentCohort = "Foxtrot 2023"
 // Expected output: "3202 tortxoF"
-
 // Pseudo code:
-
-// --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
-
+// name of function: currentCohort
+// input: 1 string
+// output: string written backwards
+// process: to seperate the string into seperate letters and then reverse the characters to create a version of the string that is written backwards.
+// const currentCohort = "Foxtrot 2023"
+// const reversedString = str.split(""){
+// }
+//     console.log(reversedString)
+var currentCohort = "Foxtrot 2023"
+console.log(currentCohort.split("").reverse)
+// NOTES: I'm not sure why the command .reverse isn't working. Again - most likely syntax issue? I thought that the command split and reverse would generate the correct output.
+// // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
+// const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+// const givenValue1 = 42
+// // Expected output: 7
+// const givenValue2 = 10
+// Expected output: 8
+// Pseudo code:
+// name of function: connects
+// input: numerical values within array
+// output: the last index of the value of the array
+// process: using lastIndexOf to assess values from the array and determine which one aligns with the last index placement
 const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+for (let i = 0; i < numberOfConnections.length; i++) {
+console.log("index", i)
+console.log("value", numsArray [i])
+}
+// NOTES: In line 166 - I initially had "i < 9" since there are 9 values in the array - but that also didn't work. The terminal says I have a syntax issue again.
+// IN CONCLUSION: None of my responses created the correct output and I need to reach out for help.
+8:56
+// 
```

Comment:
I think it might be that on line 118 there is a space between the name of the array and the square bracket. They need to be next to each other. However, since you are not using a function you have to use the exact name of the array, which is numberOfConnections.
Let's look back at the request of the prompt. It just wants the last index of the given value. Make sure that the test variables are not commented out.
Here is one approach using a built-in method .lastIndexOf()
```js
console.log(numberOfConnections.lastIndexOf(givenValue1))
console.log(numberOfConnections.lastIndexOf(givenValue2))
```